### ISSUE TITLE
[5.x]: General UI improvements for Craft 5

### ISSUE THREAD

**Issue by mikesnoeren** [craftcms/cms#16646]
### What happened?

Since the release of Craft 5, I've noticed that some accessibility features and shortcuts are either missing or have changed. Additionally, there are some general UI improvements that could enhance the user experience. Below is a list the issues I've encountered:

1. Missing Keyboard Shortcut (CMD+A) to Select All Elements
This applies to entries, categories, assets, or pretty much any element. Previously, CMD+A would select all elements in the current view, but this no longer works as expected.

1. Two Clicks Required to Remove Related Elements
We now have to open the menu and select delete, whereas it used to be a single click by simply pressing the 'x'.

1. Confusing Folder Navigation in the Assets View
When clicking a folder, the URL visibly changes, but nothing is pushed through the browser history. One might expect to use the back button on their mouse, only to be redirected to the page they were on before they started working in the asset section.

1. Inconsistent Slideout Window Buttons
Some slideout windows (like "Element settings") have a gray "Apply" button, while others (like "Field settings") have a red "Save" button. The red "Save" button can be triggered using a shortcut (CMD+S), but the "Apply" button cannot. This often leads to the browser’s "Save this webpage" window popping up.

1. Field Label Changes Don't Automatically Update the Handle
When changing the field label in the "Field settings" slideouts, you have to manually edit the handle. It would be nice if after changing the label, a message appeared saying "Create handle," and automatically filled in the handle field for you.

1. No Easy Way to Revert "Sign in as Other User"
When signed in as another user, there’s no easy option to revert back to your own admin account. Currently, you have to log out and log back in.

1. Lack of Duplicate Option for Entry Types
On the section settings screen, when choosing an Entry Type, it would be helpful to have a "Duplicate" option. We often repurpose entry types, but when they become too unique, we need to split them apart. This option would speed up that process.

1. No Option to Enable Drafts in Section Settings
While you can enable versioning for entries in a section, it would be great to also have the option to specifically enable drafts. Not all sections need draft functionality.

### Craft CMS version

5.x

### Installed plugins and versions

-

**Comment by JezDriver** [comment]
+1 for **3. Confusing Folder Navigation in the Assets View**

I frequently find myself accidentally navigating back to the wrong place when using a mouse button in asset subfolders, since the history isn't updated.

**Comment by brandonkelly** [comment]
Craft 5.8.13 is out with a fix for <kbd>Ctrl</kbd>/<kbd>Command</kbd> + <kbd>A</kbd>.
